src/main/resources/assets/amunra/lang/en_US.lang

```diff
@@ -165,18 +165,18 @@ item.thermalSuit.boots.name=Thermoregulation Boots
 item.thermalSuit.legs.name=Thermoregulation Leggings
 item.baseItem.thermalController.name=Portable Thermal Regulator
 item.baseItem.tricorder.name=Multi-Sensor
-item.baseItem.tricorder.description=Measures and displays certain data of the dimension you are in
+item.baseItem.tricorder.description=Measures and displays certain data of the dimension you are in.
 item.baseItem.tricorder.message.gravity=Gravity: %s m/s²
 item.baseItem.tricorder.message.temperature=Thermal Level: %s
 item.baseItem.tricorder.message.solar=Solar Level: %s
 item.baseItem.tricorder.message.atmosphere=Atmospheric Gasses: %s
 item.baseItem.tricorder.message.daylength=Day Length: %s
-item.baseItem.tricorder.message.unknown=Unknown
-item.baseItem.tricorder.message.unknownGas=Unknown Gas
+item.baseItem.tricorder.message.unknown=ERROR
+item.baseItem.tricorder.message.unknownGas=ERROR: gas not found.
```

Comment:
Um, yeah. I modified it to make it look like more that the tricorder tells it, more like when you go to a wrong webpage, it says 404 Error.